do/droplets.go

```diff
@@ -150,25 +156,60 @@ func (i *instances) dropletById(ctx context.Context, id string) (*godo.Droplet,
 	return droplet, nil
 }
 
+func (i *instances) dropletList(ctx context.Context) ([]godo.Droplet, error) {
+	// create a list to hold our droplets
+	list := []godo.Droplet{}
+
+	// create options. initially, these will be blank
+	opt := &godo.ListOptions{PerPage: 100}
```

Comment:
I think current droplet list limit is 200, so 100 seems safe in case Digital Ocean changes it.
Can that value be taken from a const?